analysis/interactive.Rmd

```diff
@@ -4,90 +4,259 @@ output:
 
-For analysts generating the `sce` object, we recommend saving the `sce` via one of the following methods :
+It is also possible to deploy "empty" instances of `iSEE` apps, where any `SummarizedExperiment` object stored in an RDS file may be uploaded to the running application.
+Once the file is uploaded, the application will import the `sce` object and initialize the GUI panels with the contents of the object for interactive exploration.
+This type of `iSEE` applications is launched without specifying the `sce` argument, as shown below.
 
-```{r save_rds, message=FALSE}
-# method 1
-tempfile1 <- tempfile(fileext = ".rds")
-saveRDS(sce.pbmc, tempfile1)
-# method 2
-library(HDF5Array)
-tempdir1 <- tempfile("h5_sce_")
-sce.pbmc <- saveHDF5SummarizedExperiment(sce.pbmc, tempdir1)
+```{r, eval=FALSE}
+iSEE()
 ```
 
-To read a saved `sce` object back in, you can then run:
+## Examples of usages for iSEE apps (#isee-examples)
+
+In the following subsections, we demonstrate some examples of use cases that can be addressed with interactive `iSEE` applications to gain insights into a data set and inform decision-making for downstream analyses.
+In each case, we demonstrate how `iSEE` applications can be preconfigured to launch in a specific state and layout, to facilitate immediate exploration of key aspects of the data set, in a manner most directly relevant to each situation and objectives.
+
+## Examples of usages for iSEE apps
+
+In the following subsections, we demonstrate some examples of use cases that can be addressed with interactive `iSEE` applications to gain insights into a data set and inform decision-making for downstream analyses.
+In each case, we demonstrate how `iSEE` applications can be preconfigured to launch in a specific state and layout, to facilitate immediate exploration of key aspects of the data set, in a manner most directly relevant to each situation and objectives.
+
+We note that these are examples only; we encourage readers to consider them as templates for learning and developing applications adapted to their own specific needs and purposes.
+
+### Quality control
+
+Having previously computed quality control metrics (see section [Quick start](#interactive-quickstart) above), it is possible to launch an app instance that immediately displays a set of panels preconfigured to focus on selected quality control metrics.
+
+For example, a common view for initial inspection of single-cell RNA-sequencing data sets generated by microfluidics platforms plots the library size of each cell, in decreasing order.
+An elbow in this plot generally reveals the transition between good quality cells and low quality cells or empty droplets.
+
+Another common view is to overlay the library size (in log space) as a color gradient over the result of a dimensionality reduction method (e.g., t-SNE, UMAP, ivis).
```

Comment:
Do we want to add refs to the three dim red methods?

Reply:
Done